build/platform/browser/now.js

```diff
@@ -4,5 +4,5 @@ var loadNs = performance.now();
 var loadMs = Date.now();
 
 module.exports = function () {
-  return Date.now();
-};
+  return loadMs + performance.now() - loadNs;
```

Comment:
hm, this is still going to be wrong. can you replace this entirely with 

https://github.com/lightstep/lightstep-tracer-javascript/blob/c642dd59b4ed13e8832c27345b382f5dbd2a35eb/src/imp/platform/browser/platform_browser.js#L8-L23

(in ms instead of us)

Reply:
Replaced entirely with the lightstep implementation.

Should we also update the node/now.js? https://github.com/samsarahq/dd-trace-js/blob/master/src/platform/node/now.js#L7

Reply:
the node codepath is unused